rdf-delta-integration-tests/pom.xml

```diff
@@ -110,8 +110,21 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+		  <!--
+		    Some of the test data files are not closed properly during test
+			teardown and, on Windows, cannot be deleted by subsequent test
+			setups, causing those subsequent tests to fail,
+			typically during setup.
+			A workaround is to configure the Maven Surefire plugin to execute
+			every "test suite class" in a new JVM instance; as done below.
+			As a result, all the test data files are closed when the JVM is
+			terminated at the completion of each test suite and subsequent
+			test suites can perform their setups without any problems.
+			https://github.com/afs/rdf-delta/issues/120
+		  -->
+		  <reuseForks>false</reuseForks>
           <includes>
-            <include>**/TC_*.java</include>
+            <include>Test*.java</include>
```

Comment:
The expression `**/TC_*.java` pulls in a single test suite, `TC_DeltaIntegration` that pulls in the test suites `TS_Delta` and ``TS_DeltaZk` which pull all the _real_ test classes. All those tests are executed in a single JVM instance and, as a result, cannot be divided across JVM instances.

The expression `Test*.java` pulls in the same set of _real_ test classes, but, when combined with the other change to `pom.xml` (in the lines just above this line; i.e. `<reuseForks>false</reuseForks>`) results in each test class's set of tests to be executed in a new JVM instance. Each test class seems to build its own datastore and use it for all its tests. In Windows this causes a number of problems because the datastore is not closed and cleaned up by each set of tests; so subsequent tests fail because they cannot completely clean up the old datastore. This is what I brought up in https://github.com/afs/rdf-delta/issues/120 . By shutting down and restarting the JVM between each set of tests, all the files in the datastore are _closed_ (as a result of the JVM shutting down), allowing the subsequent set of tests to clean up any leftover files without a problem in a new JVM instance. See the comment in `pom.xml` lines 113-124.

This does not fix the root problem of how the tests do not properly close their datastores and the associated files. It is simply a reasonable workaround.